lib/src/usage.dart

```diff
@@ -122,6 +122,8 @@ class _Usage {
     var result;
     if (option.negatable!) {
       result = '--[no-]${option.name}';
+    } else if (option.mandatory) {
+      result = '--${option.name}';
```

Comment:
I don't understand what this condition is accomplishing. Is this supposed to be different from the block below?

Reply:
This is a mistake, sorry !